first_ruby/Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
```

Comment:
Whats this file for?

Reply:
This file is to locate and load tasks which can be run from the command line. Which also known to help building, packaging and testing of Rails code.

Reply:
I don't understand what you mean. Give example please

Reply:
Rake is going to execute task which are defined in `Rakefile` & file in directory `lib/tasks` ending in `.rake`. The example of commands can be `rake restart` to restart app, to get all the rake task, type `rake -T` in the console and it will show all the task along with the description.

Reply:
Ok. So, what is `rake`?

Reply:
`rake` is a task runner used by rails